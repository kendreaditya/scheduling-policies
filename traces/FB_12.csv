1,0,50
2,10,40
3,20,30
4,30,20
5,40,10
6,50,0
7,60,10
8,70,20
9,80,30
10,90,40
11,100,50
12,110,60
13,120,70
14,130,80
15,140,90
16,150,100
17,160,110
18,170,120
19,180,130
20,190,140
21,600,437
22,600,437
23,1000,0
24,1000,1
25,1000,2
26,1000,3
27,1000,4
28,1000,5
29,1000,6
30,1000,7
