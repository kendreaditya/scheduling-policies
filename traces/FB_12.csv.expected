1,850
2,720
3,510
4,360
5,50
6,50
7,70
8,360
9,510
10,720
11,850
12,960
13,1088
14,1178
15,1258
16,1328
17,1388
18,1438
19,1478
20,1508
21,2102
22,2102
23,1000
24,1007
25,1013
26,1018
27,1022
28,1025
29,1027
30,1028
